azure/template.json

```diff
@@ -0,0 +1,183 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "serviceName": {
+            "type": "string"
+        },
+        "appServicePlanResourceGroup": {
+            "type": "string"
```

Comment:
should have a default value property that is set to empty

defaultValue: ""

or is this parameter event needed?

Reply:
replaced with [resourceGroup().name]